components/Navbar/Navbar.tsx

```diff
@@ -24,6 +79,14 @@ const Navbar = () => {
       ? 'hover:bg-orange-700'
       : 'hover:bg-gray-700'
 
+  const [showNotifications, setShowNotifications] = useState<boolean>(false)
+
+  const [hasNotifications, setHasNotifications] = useState<boolean>(true)
+
+  const handleToggleNotifications = () => {
```

Comment:
nitpick: no need to add `handle` in the beginning of a function with a clear and explicit name like `toggleNotifications`